centrallix/netdrivers/net_http_rest.c

```diff
@@ -115,8 +115,8 @@ nht_i_RestWriteAttrValue(pNhtConn conn, pObject obj, char* attrname, int data_ty
 
 	    case DATA_T_MONEY:
 		nht_i_QPrintfConn(conn, 0, "{ \"wholepart\":%INT, \"fractionpart\":%INT }",
-			od.Money->WholePart,
-			od.Money->FractionPart
+			od.Money->Value/10000,
+			od.Money->Value%10000
```

Comment:
Once the %LL specifier is defined, this issue will resolve, correct?  Because after that, I should change the arguments to "'moneyValue: %LL', od.Money->Value".  Unless we want to keep the wholepart/fractionpart split for nht_i_QPrintfConn.

Reply:
I would like for us to migrate the JSON REST API to using a single numeric value, but we have to make some design decisions first with that.  For now, the JSON data still should have the whole/fraction split, and it should keep the old representation standard of the fraction part always moving the value in the positive direction.